src/ContaoCommunityAlliance/DcGeneral/Contao/View/Contao2BackendView/Widget/FileTreeOrder.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * PHP version 5
+ *
+ * @package    generalDriver
+ * @author     David Molineus <david.molineus@netzmacht.de>
+ * @copyright  The MetaModels team.
+ * @license    LGPL.
+ * @filesource
+ */
+
+namespace ContaoCommunityAlliance\DcGeneral\Contao\View\Contao2BackendView\Widget;
+
+/**
+ * This widget is a supporting widget to store the file tree orderings.
+ *
+ * The ContaoWidgetManager does not allow input values without a widget. This is used as helper widget instead.
+ */
+class FileTreeOrder extends AbstractWidget
+{
+    /**
+     * The template.
+     *
+     * @var string
+     */
+    protected $strTemplate = 'be_widget_rdo';
```

Comment:
In general I agree with you. In this case I doubt there would be any changes as the history shows us:

https://github.com/contao/core/commits/master/system/modules/core/templates/backend/be_widget_rdo.html5

Well, I use a custom template and move the html in there.